src/services/F2fService.ts

```diff
@@ -379,4 +379,24 @@ export class F2fService {
 		}
 	}
 
+	async updateSessionTtl(sessionId: string, sessionTtl: number, tableName: string = this.tableName): Promise<void> {
```

Comment:
think unit test when `updateSessionTtl` fails is missing in F2FService.test.ts with asserts on log messages and error code